src/olympia/addons/models.py

```diff
@@ -625,6 +625,12 @@ def resolve_webext_translations(cls, data, upload):
         This returns a modified `data` dictionary accordingly with proper
         translations filled in.
         """
+        # If we've already done this then data[xxx] will be dict already.
```

Comment:
fwiw, this codepath isn't used any more (because I moved the akismet check to the file_upload validation instead) so... delete?  (+test)

Reply:
If it's not used, let's get rid of it then

Reply:
https://github.com/mozilla/addons-server/pull/9353/commits/21cf8539585693743eeda3307be76faba7b31ba9